photon-client/src/components/dashboard/tabs/TargetsTab.vue

```diff
@@ -80,14 +110,16 @@ const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings
         </template>
       </v-simple-table>
     </v-row>
-    <v-row
+    <template
```

Comment:
This shouldn't be a template, if you don't want the `v-row` styling use a div

Reply:
I want the if to apply to both, tho